efficientdet/keras/train_lib.py

```diff
@@ -386,6 +387,32 @@ def get_callbacks(params):
   return callbacks
 
 
+class AdversarialLoss(tf.keras.losses.Loss):
+  """Adversarial keras loss wrapper"""
+  #TODO(fsx950223): WIP
+  def __init__(self, adv_config, *args, **kwargs):
+    super().__init__(*args, **kwargs)
+    self.adv_config = adv_config
+    self.model = None
+    self.loss_fn = None
+    self.tape = None
+    self.built=False
+
+  def build(self, model, loss_fn, tape):
+    self.model = model
+    self.loss_fn = loss_fn
+    self.tape = tape
+    self.built = True
+
+  def call(self, features, y, y_pred, labeled_loss):
+    return self.adv_config.multiplier * nsl.keras.adversarial_loss(features,
+                               y,
```

Comment:
This library looks cool.

BTW, the indent looks a little strange. Could you also format the code with yapf?